doc.go

```diff
@@ -22,6 +22,21 @@ of sub-packages.
 All clients in sub-packages are configurable via client options. These options are
 described here: https://godoc.org/google.golang.org/api/option.
 
+## Endpoint
+
+Endpoint configuration is used to specify the API endpoint to which requests are
+sent. It is primarily used for services that support or require regional endpoints.
+For example, the Vertex AI service recommends that you set the location with the
+features you want that is closest to your physical location or the physical location
+of your users. There is no global endpoint for Vertex AI. See
+[Vertex AI - Locations](https://cloud.google.com/vertex-ai/docs/general/locations)
+for more details. The following example demonstrates configuring a Vertex AI client
+with a regional endpoint:
+
+	ctx := context.Background()
+	endpoint := "us-central1-aiplatform.googleapis.com"
```

Comment:
I believe gRPC clients require the port.

Reply:
Fixed. Ooooops! Should I add this to the [WithEndpoint](https://pkg.go.dev/google.golang.org/api/option#WithEndpoint) docs??? Right now they say nothing about the `url` param.

> // WithEndpoint returns a ClientOption that overrides the default endpoint
> // to be used for a service.